### PR TITLE
Tabla sound font

### PR DESCRIPTION
**Pull request by shivrajsa**
Tabla is a popular Indian percussion instrument.
Here are MIDI.js compatible format of Tabla.sf2

Not all notes contains sound, use sound font software like Viena to find details of sound and MIDI key

### CODE REVIEW

File: Tabla-soundfont/tabla-mp3/file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+dummy file to create folder
```
Comment (gleitz) [inline Tabla-soundfont/tabla-mp3/file.txt]: This can be removed

**Comment by gleitz** [comment]
Awesome! Would you please update the README with the new soundfont info? You can use `Fluid-Soundfont` as a template.

File: README.md [hunk 2]
```diff
@@ -20,11 +20,17 @@ Soundfonts Available
     - URL prefix to fetch files: http://gleitz.github.io/midi-js-soundfonts/MusyngKite/
 
 - FatBoy Soundfont
-    - Generated from [FatBoy.sf2](https://fatboy.site) (330 MB uncompressed)
+    - Generated from [FatBoy.sf2](https://github.com/shivrajsa/midi-js-soundfonts/blob/gh-pages/Tabla-soundfont/Tabla.sf2) (4.06 MB uncompressed)
```
Comment (gleitz) [inline README.md]: This line should not have changed - only the lines below.

File: README.md [hunk 3]
```diff
@@ -20,11 +20,17 @@ Soundfonts Available
...
     - Released under [Creative Commons Attribution Share-Alike 3.0 license](https://creativecommons.org/licenses/by-sa/3.0/)
     - Instrument names as .json file [here](http://gleitz.github.io/midi-js-soundfonts/FatBoy/names.json)
     - URL prefix to fetch files: http://gleitz.github.io/midi-js-soundfonts/FatBoy/
 
+- Tabla-Soundfont
+    - Tabla is a popular Indian percussion instrument. Not all notes contains sound in Tabla.sf2, sounds are mapped on notes C4 to E6. Use sound font software like [Viena](http://www.synthfont.com/index.html) to find details of each sound and MIDI key.
+    - Generated from Tabla.sf2 (4.06 MB uncompressed)
```
Comment (gleitz) [inline README.md:30]: I wonder if we can link to the SF2 file if it's hosted anywhere

File: README.md [hunk 4]
```diff
@@ -20,11 +20,17 @@ Soundfonts Available
...
+    - Instrument name : Tabla is not standard MIDI instrument, you need to map it to appropriate instrument in your program.
```
Comment (gleitz) [inline README.md]: Perhaps a code snippet on how to get the tabla to work?

**Comment by shivrajsa** [comment]
My first accepted pull request on Github. :-)
Thank you @gleitz

**Comment by gleitz** [comment]
Nice work!

**Pull request merged**